src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+
+public abstract class AbstractManager extends AbstractEmployee{
+
+    ArrayList<AbstractEmployee> employees_list;
+
+    AbstractManager(String name, double salary, ArrayList<AbstractEmployee> employees_list) {
+        super(name, salary);
+        this.employees_list = new ArrayList<>();
+    }
+
+    ArrayList<AbstractEmployee> getEmployees_list() {
+        return employees_list;
+    }
+
+    public void setEmployees_list(ArrayList<AbstractEmployee> employees_list) {
+        this.employees_list = employees_list;
+    }
+
+
+    public String toStr(int line) {
+        String result = new String();
+        for(int i=0;i<line;i+=1)
```

Comment:
IntelliJ has a shortcut (cmd + alt + L on MacOS, something similar on Windows/Linux) to reformat the code and put spaces where they should be :)